scripts/_vault_commands.py

```
@@ -0,0 +1,23 @@
+"""Helpers for interacting with HashiCorp Vault during bootstrap."""
+
+from __future__ import annotations
+
+import json
+from typing import Any, Dict
```

Ruff (0.14.1)

`typing.Dict` is deprecated, use `dict` instead

(UP035)